README.md

```diff
@@ -0,0 +1,90 @@
+# Relativity
+
+[![CI Status](https://travis-ci.org/dfed/Relativity.svg?branch=master)](https://travis-ci.org/dfed/Relativity)
+[![Carthage Compatibility](https://img.shields.io/badge/carthage-✓-e2c245.svg)](https://github.com/Carthage/Carthage/)
+[![Version](https://img.shields.io/cocoapods/v/Relativity.svg)](http://cocoadocs.org/docsets/Relativity)
+[![License](https://img.shields.io/cocoapods/l/Relativity.svg)](http://cocoadocs.org/docsets/Relativity)
+[![Platform](https://img.shields.io/cocoapods/p/Relativity.svg)](http://cocoadocs.org/docsets/Relativity)
+
+A DSL for laying out views without Auto Layout in Swift.
```

Comment:
They don't take up too much real estate 😝. But you're def right: I need to buff out this section a bit. Probably goes hand-in-hand with write a blog post.